tests/run_tests.sh

```diff
@@ -13,6 +13,11 @@ pip_install() {
     cd /root/gentoo_update_source && pip install . --break-system-packages
 }
 
+pypi_install() {
+    # Install with pip from PyPi
+    pip install gentoo_update --break-system-packages
```

Comment:
Can you add a comment on why we need to break-system-packages?

Reply:
Becuase [PEP668](https://peps.python.org/pep-0668/) changed how `pip` installs packages on the system.
This is a script to test gentoo_update in a container, I don't think it matters very much how it's being installed, since container will get destroyed afterwards.
But I have already updated the [readme](https://github.com/Lab-Brat/gentoo_update#usage) to install it in Python's virtual environment instead of using `--break-system-packages` flag.